be/src/util/thrift_rpc_helper.cpp

```diff
@@ -59,6 +59,36 @@ void ThriftRpcHelper::setup(ExecEnv* exec_env) {
     _s_exec_env = exec_env;
 }
 
+template <typename T>
+Status ThriftRpcHelper::rpc_impl(std::function<void(ClientConnection<T>&)> callback, ClientConnection<T>& client,
+                                 const TNetworkAddress& address) noexcept {
+    std::stringstream ss;
+    try {
+        callback(client);
+    } catch (apache::thrift::protocol::TProtocolException& e) {
+        if (e.getType() == apache::thrift::protocol::TProtocolException::TProtocolExceptionType::INVALID_DATA) {
+            ss << "FE RPC response parsing failure, address=" << address
+               << ". The FE may be busy, please retry later or increase BE config thrift_rpc_timeout_ms";
```

Comment:
not all rpcs use BE config `thrift_rpc_timeout_ms`

Reply:
Maybe it's better to be less specific.